doc/source/rllib/package_ref/offline.rst

```diff
@@ -0,0 +1,61 @@
+.. _offline-reference-docs:
+
+Offline RL
+==========
+
+InputReaders
+------------
+
+The InputReader API is used by individual :py:class:`~ray.rllib.evaluation.rollout_worker.RolloutWorker`s
```

Comment:
didn't render this properly:
![Uploading Screen Shot 2021-12-06 at 8.45.53 PM.png…]()

Reply:
fixed